components/prism-sqf.js

```diff
@@ -0,0 +1,33 @@
+Prism.languages.sqf = Prism.languages.extend('clike', {
+	'string': {
+		pattern: /"(?:(?:"")?[^"])*"(?!")|'(?:[^'])*'/,
+		greedy: true
+	},
+	'keyword': /\b(?:call|case|catch|default|do|echo|else|endsWith|for|forEach|from|goto|if|nil|private|step|switch|then|to|try|while)\b/,
```

Comment:
Should also move `with` to "keyword" list (from "function" list). Sublime-SQF-Language considers it a keyword.

Reply:
Agreed.